### PR TITLE
Adding yuidoc - Closes #15

### LINKED ISSUE #15: Document source code using YUIDoc
We should document the `lisk-js` source code using: http://yui.github.io/yuidoc/

New code can be documented immediately: http://yui.github.io/yuidoc/syntax/index.html

Existing code can we cover here, module by module.

### PR DESCRIPTION
**Pull request by Tosch110**
Adding yuidoc as mentioned in #15

### CODE REVIEW

File: lib/time/slots.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
+/**
+ * Slots module helping with dates and times in Lisk blockchain
```
Comment (Isabello) [inline lib/time/slots.js]: Slots module provides helpers for dates and times for the Lisk Blockchain

File: lib/transactions/crypto.js [hunk 2]
```diff
@@ -1,24 +1,44 @@
-var crypto = require("crypto-browserify");
-var constants = require("../constants.js");
+/**
+ * Crypto module helping with cryptograhpy in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/crypto.js]: Crypto module provides functions for cryptographic functions in the Lisk Blockchain

File: lib/transactions/dapp.js [hunk 3]
```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating dapps in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/dapp.js]: Dapps module provides functions used to create dapps (blockchain apps) for the Lisk Blockchain

File: lib/transactions/delegate.js [hunk 4]
```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating delegates in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/delegate.js]: Delegates module provides functions to create delegates for the Lisk Blockchain.

File: lib/transactions/multisignature.js [hunk 5]
```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating multisignature in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/multisignature.js]: Multisignatures module provides functions for creating multisignature transactions for use with the Lisk Blockchain

File: lib/transactions/signature.js [hunk 6]
```diff
@@ -1,6 +1,18 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating signature in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/signature.js]: Signatures module provides functions for creating signatures for use with the Lisk Blockchain.

File: lib/transactions/transaction.js [hunk 7]
```diff
@@ -1,6 +1,21 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating transactions in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/transaction.js]: Transactions module provides functions for creating transactions for use with the Lisk Blockchain.

File: lib/transactions/transfer.js [hunk 8]
```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating dapp transfer in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/transfer.js]: Transfer module provides functions for use with DAPP (blockchain apps?) transfers to the Lisk Blockchain.

File: lib/transactions/vote.js [hunk 9]
```diff
@@ -1,6 +1,20 @@
-var crypto = require("./crypto.js"),
-    constants = require("../constants.js"),
-    slots = require("../time/slots.js");
+/**
+ * Crypto module helping with creating votes in Lisk Blockchain
```
Comment (Isabello) [inline lib/transactions/vote.js]: Vote module provides functions for creating vote transactions for use with the Lisk Blockchain.

**Comment by Tosch110** [comment]
Changed class summaries

**Pull request merged**